docs/Server.md

```diff
@@ -473,6 +473,19 @@ const fastify = require('fastify')({
 })
 ```
 
+<a name="rewrite-url"></a>
+### `rewriteUrl`
+
+Set a sync callback function that must return a string that allows rewriting urls. 
```

Comment:
Can you add a note that this function cannot be encapsulated and thus it's a global configuration?